fury/stream/servers/webrtc/async_app.py

```diff
@@ -26,6 +27,9 @@ async def javascript(request, **kwargs):
 
 async def offer(request, **kwargs):
     video = kwargs['video']
+    if("broadcast" in kwargs and kwargs["broadcast"]):
+        video = MediaRelay().subscribe(video)
```

Comment:
I think that it's not the best approach because creates a new MediaRelay instance for each offer call.
See for example
https://github.com/abhiTronix/vidgear/blob/eb19b9ce96061c09b5fe4c3ee63d84c940a19af2/vidgear/gears/asyncio/webgear_rtc.py#L493